Microsoft.Toolkit.Uwp.UI/Cache/ImageCache.cs

```diff
@@ -54,9 +58,9 @@ public class ImageCache : CacheBase<BitmapImage>
         /// <returns>awaitable task</returns>
         protected override async Task<BitmapImage> InitializeTypeAsync(StorageFile baseFile)
         {
-            using (var stream = await baseFile.OpenReadAsync().AsTask().ConfigureAwait(false))
+            using (var stream = await baseFile.OpenReadAsync())
```

Comment:
Can probably stick `.ConfigureAwait(false)` here given the below uses it as well